openapi/openapi.yaml

```diff
@@ -194,6 +194,12 @@ paths:
           required: true
           schema:
             type: string
+        - name: managingCluster
+          in: query
+          description: If specified, which managing cluster to create kube-api access for ( service | management | hive )
```

Comment:
If we have a predefined list of valid values, we should use an enum https://swagger.io/docs/specification/v3_0/data-models/enums/